dart/lib/src/transport/http_transport.dart

```diff
@@ -16,15 +17,15 @@ class HttpTransport implements Transport {
   final Dsn _dsn;
 
   /// Use for browser stacktrace
-  final String _origin;
+  //final String _origin;
 
   CredentialBuilder _credentialBuilder;
 
   final Map<String, String> _headers;
 
-  HttpTransport({@required SentryOptions options, String origin})
+  HttpTransport({@required SentryOptions options /*, String origin*/})
       : _options = options,
-        _origin = origin,
+        /*_origin = origin,*/
```

Comment:
clean up the comments if they are not relevant

Reply:
✅